src/pages/FormPage.js

```diff
@@ -6,7 +6,9 @@ export default function FormPage({ handleData }) {
   return (
     <FormPageStyle>
       <Header>
-        <h1>Hier kannst Du eintragen, wie es mir schmeckt...</h1>
+        <h1>
+          Hier kannst Du eintragen, <br /> wie es mir schmeckt...
```

Comment:
Do you really want to set the break manually? Maybe the `word-break` CSS property is what you need here?

Reply:
Thanks for mention this and the css advice. I added the manual break after my own quality assurance on the mobile for structuring the headline always in this way. Checking the `word-break`CSS property it doesn't seem to me that it will do the job like this.